Clients/GUI/Pages/Instruments.razor

```diff
@@ -125,9 +126,11 @@
         }
     }
 
-    private void SaveInstrument(Instrument instrument)
+    private void GoToInstrumentPage(Instrument instrument)
     {
-
+        var path = "/instrument";
```

Comment:
Can you please tell me which page we're going to go to?

Reply:
fixed. Now it is obvious